src/test/java/io/quarkus/bot/it/CheckIssueEditorialRulesTest.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.bot.it;
+
+import java.io.IOException;
+import static org.mockito.Mockito.*;
+
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GHEvent;
+
+import io.quarkiverse.githubapp.testing.GitHubAppTestingResource;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+
+import static io.quarkiverse.githubapp.testing.GitHubAppTesting.when;
+
+@QuarkusTest
+@QuarkusTestResource(GitHubAppTestingResource.class)
+public class CheckIssueEditorialRulesTest {
+    @Test
+    void validZulipLinkConfirmation() throws IOException {
+        when().payloadFromClasspath("/issue-opened-zulip.json")
+                .event(GHEvent.ISSUES)
+                .then().github(mocks -> {
+                    verify(mocks.issue(942074921))
+
+                            .comment(
+                                    "You added a link to a Zulip discussion, please make sure the description of the issue is comprehensive and doesn't require accessing Zulip."
+                                            + "\n\n> This message is automatically generated by a bot.");
```

Comment:
Please use the `ZULIP_WARNING` constant here, we made it public for exactly this reason.